js/quicksort.js

```diff
@@ -14,22 +15,23 @@ function quickSort(arr) {
   }
 
   let pivot = arr[0];
```

Comment:
On top of keeping track of the value of the pivot, you should be removing it from the array so that it doesn't get processed twice.